### PR TITLE
fix: move repo README content to oa-docs

### PR DESCRIPTION
**Pull request by ryanseee**
Moving of contents in README of different repositories to oa-docs

wip:
- Open Attestation (Verify)

### CODE REVIEW

File: website/docs/developer/decentralized-renderer.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+id: decentralized-renderer
+title: Decentralized Renderer
+sidebar_label: Decentralized Renderer
+---
+
+Overview
+
+## Header
+
+TBD
```
Comment (simboonlong) [inline website/docs/developer/decentralized-renderer.md:11]: there are example template renderers if users want to kick start their own, comes in 3 flavours:

- https://github.com/Open-Attestation/decentralized-renderer-react-template
- https://github.com/Open-Attestation/decentralized-renderer-svelte-template
- https://github.com/Open-Attestation/decentralized-renderer-vue-template

maybe can write 1 or 2 liner to point them to the respective repos

File: website/docs/developer/document-store.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+- [Deploy (with minimal proxy)](#deploy-with-minimal-proxy)
+- [Interact](#interact)
+- [List of available functions](#list-of-available-functions)
+- [Provider & Signer](#provider--signer)
+- [Setup](#setup)
+- [Contract Benchmark](#contract-benchmark)
+- [Notes](#notes)
+
+### Deploy
+
```
Comment (simboonlong) [inline website/docs/developer/document-store.md:34]: copy the titles? `Deploying new document store` ? thought they were more descriptive. same with the rest

File: website/docs/developer/open-attestation-cli.md [hunk 3]
```diff
@@ -0,0 +1,677 @@
...
+- [Wrapping document](#wrapping-documents)
+- [Document privacy filter](#document-privacy-filter)
+- [Encrypting document](#encrypting-document)
+- [Decrypting document](#decrypting-document)
+- [Token registry](#token-registry)
+  - [Deploy](#deploy)
+  - [Issue](#issue)
+- [Document store](#document-store)
+  - [Deploy](#deploy-1)
+  - [Issue](#issue-1)
```
Comment (simboonlong) [inline website/docs/developer/open-attestation-cli.md:50]: nice these are in their own respective headers now, able to be explicit?
```suggestion
- [Token registry](#token-registry)
  - [Deploy](#deploy-token-registry)
  - [Issue](#issue-token-registry)
- [Document store](#document-store)
  - [Deploy](#deploy-document-store)
  - [Issue](#issue-document-store)
```

File: website/docs/developer/document-store.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+## Contract Benchmark
+
+A benchmark is provided to show the different transaction cost of the different variants of the document store.
+
+```sh
+npm run benchmark
+```
+
+![Benchmark Results](../../static/docs/component/document-store/benchmark.png)
+
```
Comment (simboonlong) [inline website/docs/developer/document-store.md:135]: benchmark might potentially go outdated without us knowing (code changed, then forget to update this screenshot) i prefer not to put the screenshot here

File: website/docs/developer/open-attestation-cli.md [hunk 5]
```diff
@@ -0,0 +1,677 @@
...
+
+To run on local for development
+
+```
+npm run dev -- <command> <options>
+```
+
+## Test
+
+```
```
Comment (simboonlong) [inline website/docs/developer/open-attestation-cli.md:649]: this part onwards stays at readme?

File: website/docs/developer/open-attestation-encryption.md [hunk 6]
```diff
@@ -0,0 +1,203 @@
+---
+id: open-attestation-encryption
+title: Open Attestation (Encryption)
+sidebar_label: Open Attestation (Encryption)
+---
+
+The [Open Attestation (Encryption)](https://github.com/Open-Attestation/oa-encryption) repository is the codebase for the npm module that allows you ti encrypt documents in order to share and store them safely.
+
```
Comment (simboonlong) [inline website/docs/developer/open-attestation-encryption.md:8]:
```suggestion
The [Open Attestation (Encryption)](https://github.com/Open-Attestation/oa-encryption) repository is the codebase for the npm module that allows you to encrypt documents in order to share and store them safely.

```

File: website/docs/developer/overview.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+id: overview
+title: Overview
+sidebar_label: Overview
+---
+
+OpenAttestation comes with a set of tools that help you to build, verify or share documents depending on your need.
+
+In this section you will discover all the tools that OpenAttestation offers and how to use them:
+
+- [Wrapper SDK (javascript)](/docs/developer/open-attestation) is the core of OA allowing you to create transferable record or verifiable document.
```
Comment (simboonlong) [inline website/docs/developer/overview.md:11]: should we rename the links as what the titles is?
```suggestion
- [open-attestation (javascript)](/docs/developer/open-attestation) is the core of OA allowing you to create transferable record or verifiable document.
```
same for the rest

File: website/docs/developer/token-registry.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+// Signer
+const signerFromPrivateKey = new Wallet("YOUR-PRIVATE-KEY-HERE", provider);
+const signerFromEncryptedJson = Wallet.fromEncryptedJson(json, password);
+signerFromEncryptedJson.connect(provider);
+const signerFromMnemonic = Wallet.fromMnemonic("MNEMONIC-HERE");
+signerFromMnemonic.connect(provider);
+```
+
+### Setup
+
```
Comment (simboonlong) [inline website/docs/developer/token-registry.md:111]: this onwards stays at readme?

File: website/docs/developer/open-attestation.md [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+
+`obfuscateDocument` removes a key-value pair from the document's data section, without causing the file hash to change. This can be used to generate a new document containing a subset of the original data, yet allow the recipient to proof the provenance of the document.
+
+```js
+const newData = obfuscateDocument(wrappedDocument, "key1");
+console.log(newData);
+```
+
+## Test
+
```
Comment (simboonlong) [inline website/docs/developer/open-attestation.md:176]: this part onwards stays at readme?

File: website/docs/developer/document-store.md (hunk 2 above)
Comment (ryanseee) [reply website/docs/developer/document-store.md]: ok done.

File: website/docs/developer/open-attestation-cli.md (hunk 3 above)
Comment (ryanseee) [reply website/docs/developer/open-attestation-cli.md]: ok done.

File: website/docs/developer/document-store.md (hunk 4 above)
Comment (ryanseee) [reply website/docs/developer/document-store.md]: ok done.

File: website/docs/developer/open-attestation-encryption.md (hunk 6 above)
Comment (ryanseee) [reply website/docs/developer/open-attestation-encryption.md]: ok done.

File: website/docs/developer/overview.md (hunk 7 above)
Comment (ryanseee) [reply website/docs/developer/overview.md]: ok done.

File: website/docs/developer/open-attestation.md [hunk 10]
```diff
@@ -0,0 +1,217 @@
...
+const newData = obfuscateDocument(wrappedDocument, "key1");
+console.log(newData);
+```
+
+## Development
+
+To run on local
+
+```
+npm run dev -- <command> <options>
```
Comment (Nebulis) [inline website/docs/developer/open-attestation.md:181]: nope, not in this repo :D

File: website/docs/developer/overview.md [hunk 11]
```diff
@@ -0,0 +1,16 @@
...
+
+OpenAttestation comes with a set of tools that help you to build, verify or share documents depending on your need.
+
+In this section you will discover all the tools that OpenAttestation offers and how to use them:
+
+- [Open Attestation](/docs/developer/open-attestation) is the core framework that allows you to create OA documents such as, transferable record or verifiable document.
+- [Open Attestation (Verify)](/docs/developer/open-attestation-verify) helps you to verify that documents are valid OA documents.
+- [Open Attestation (Encryption)](/docs/developer/open-attestation-encryption) assists you to share and store your OA documents in a secure way.
+- [Decentralized Renderer](/docs/developer/decentralized-renderer) allows recipient of OA documents to view their document in a human readable format.
+
```
Comment (simboonlong) [inline website/docs/developer/overview.md:15]: wait now i've realise this is the same content as `Components` overview, can we write something different here, like explaining the need/overview for this `Developer` section

**Pull request closed without merging**